tests/golden/check-footnotes/check-footnotes.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "We report broken links inside footnotes" {
```

Comment:
The second test in this file seems redundant. More specifically, if the 1st test passes, the 2nd test must also pass.

If we want to test these two behaviours separately (which I think is a good idea), we could move the two markdown files into two separate folders, and then the first test would run `xrefcheck -r <first_dir>` and the second test would run `xrefcheck -r <second_dir>`

Reply:
Done